database/tables/variants.pg

```diff
@@ -1,6 +1,7 @@
 columns
     authn_user_id: bigint
     broken: boolean default false
+    course_id: bigint
```

Comment:
Add a NOT NULL constraint.

Reply:
We can't do that yet because this column won't be populated yet; that's the whole point of the batched migration machinery I've been working on!

Reply:
Ugh, right, thanks for the reminder yet again! 🤦

Reply:
to be fair, this PR still has the code to populate the column, so it was easy to be confused (I'll be ripping that out today).